staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -154,6 +154,7 @@ func NewCmdExposeService(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	cmd.Flags().String("container-port", "", i18n.T("Synonym for --target-port"))
 	cmd.Flags().MarkDeprecated("container-port", "--container-port will be removed in the future, please use --target-port instead")
 	cmd.Flags().String("target-port", "", i18n.T("Name or number for the port on the container that the service should direct traffic to. Optional."))
+	cmd.Flags().String("node-port", "", i18n.T("Port used to expose the service on each node in a cluster."))
```

Comment:
From what I see `kubectl expose` is following the path that we stopped `kubectl run` from. Iow. it's growing its functionality beyond basic capabilities. There is `kubectl create nodeport` for advanced use cases and I'd prefer we don't support all possibilities here.

Reply:
Additionally, we're on a path to get rid of generators of any flavor so this PR would be stomping that effort along the way too.

Reply:
> From what I see kubectl expose is following the path that we stopped kubectl run from.

I have no problem with closing this PR.
Are there any documents that I should read before I start development that indicate such a development policy?